pulser-core/pulser/register/base_register.py

```diff
@@ -206,7 +206,9 @@ def _validate_layout(
                 )
 
     def define_detuning_map(
-        self, detuning_weights: Mapping[QubitId, float]
+        self,
+        detuning_weights: Mapping[QubitId, float],
+        slug: str | None = None,
```

Comment:
The slug is missing a description in the docstring (here and in other `define_detuning_map` methods.